### DISCUSSION TITLE
gRPC Support

### DISCUSSION THREAD

**Discussion by marclave** [scalar/scalar#7794] [Ideas]
eventually!

**Comment by byjokese** [comment]
Interested on this one!

**Comment by cwevers** [comment]
Yes please

**Comment by XenayaDev** [comment]
Would really love to see this feature, are there any news on this?

**Comment by heshamMassoud** [comment]
Any updates on this? this would be fantastic to have on scalar.

**Comment by tmp27017** [comment]
any update :)
